build/generate-release-list.js

```diff
@@ -0,0 +1,21 @@
+const octokit = require('@octokit/rest')();
+const fs = require('fs');
+
+const list = {};
+
+octokit
+    .paginate(octokit.repos.listReleases.endpoint({
```

Comment:
Does this need to get _all_ the releases? The dropdown to pick previous versions is nice, but in my ad-hoc testing I found the some of the examples do not work on older releases because the style or runtime functionality are not supported. 
A simple alternative would be to use URL query parameters to specify the version.

Reply:
It doesn't _have_ to get all releases; we could e.g. limit to the last 20 releases or so. Parsing the version from the dropdown works, but I found it convenient when testing on mobile devices.